sdc/datatypes/hpat_pandas_series_functions.py

```diff
@@ -397,6 +397,30 @@ def hpat_pandas_series_values_impl(self):
 @overload_method(SeriesType, 'value_counts')
 def hpat_pandas_series_value_counts(self, normalize=False, sort=True, ascending=False, bins=None, dropna=True):
     """
+    Intel Scalable Dataframe Compiler User Guide
+    ********************************************
+
+    Pandas API: pandas.Series.value_counts
+
+    Examples
+    --------
+    .. literalinclude:: ../../../examples/series/value_counts.py
+       :language: python
+       :lines: 27-
+       :caption: Getting the number of not NaN values occurrence
+       :name: ex_series_value_counts
+
+    .. code-block:: console
+
+        > python ./series/value_counts.py
+        3.0    2
+        4.0    1
+        2.0    1
+        1.0    1
+        dtype: int64
+
```

Comment:
Missing See Also section - I suppose the count() method is relevant here

Reply:
done